lib/src/widgets/rectangle_animated_snack_bar.dart

```diff
@@ -3,73 +3,101 @@ import 'package:flutter/material.dart';
 import '../types.dart';
 
 const _infoBackgroundColor = Color.fromRGBO(85, 85, 238, 1);
-const _errorBackgrounColor = Color.fromRGBO(255, 0, 0, 1);
+const _errorBackgroundColor = Color.fromRGBO(255, 0, 0, 1);
 const _successBackgroundColor = Color.fromRGBO(99, 142, 90, 1);
 const _warningBackgroundColor = Color.fromRGBO(232, 145, 72, 1);
 
 const _darkColor = Color.fromRGBO(45, 45, 40, 1);
 
+const _infoIcon = Icons.info_outline;
+const _errorIcon = Icons.error_outline;
+const _successIcon = Icons.check_circle_outline;
+const _warningIcon = Icons.warning_amber_outlined;
+
 class RectangleAnimatedSnackBar extends StatelessWidget {
   const RectangleAnimatedSnackBar({
     Key? key,
     required this.titleText,
     required this.messageText,
     required this.type,
     required this.brightness,
+    this.infoBackgroundColor = _infoBackgroundColor,
+    this.errorBackgroundColor = _errorBackgroundColor,
+    this.successBackgroundColor = _successBackgroundColor,
+    this.warningBackgroundColor = _warningBackgroundColor,
+    this.infoIcon = _infoIcon,
```

Comment:
For this part, Why don't just get one icon and one color? Or use factories for each type

Reply:
I wasn't able to continue with it.
Apologies for the delay.